ibc-testkit/src/hosts/mod.rs

```diff
@@ -119,25 +122,35 @@ pub trait TestHost: Default + Debug + Sized {
         }
         Ok(())
     }
+
+    fn header_params<C>(
+        &self,
+        client_id: &ClientId,
+        client_context: &C,
+    ) -> HostLightClientHeaderParams<Self>
+    where
+        C: ClientValidationContext;
```

Comment:
Introduced this to generalize the parameters needed to convert a block to a light client header.

Reply:
To be honest, I found this new method a bit confusing and difficult to grasp its purpose. Since we have tight plans, I made some simplifications ([410b8ad](https://github.com/cosmos/ibc-rs/pull/1146/commits/410b8adf535c6247b6f8cc90ab869edf6b61b65f)) so that can move forward and wrap up ibc-testkit refactoring. I hope that works for you!